src/app/pages/Home/OtherComponents/MainBanner.module.css

```diff
@@ -0,0 +1,3 @@
+.popup-address-chip {
+  font-size: 0.75rem !important;
```

Comment:
could we avoid !imporant?

Reply:
Sorry, we need it because here https://github.com/madfish-solutions/templewallet-extension/blob/TW-785-transaction-history-send-receive-extension/src/app/atoms/CopyButton.tsx#L75 are some classes for font size but we avoid using small property to keep the size of the chip